docs/tutorials/misc/300_otcp.ipynb

```diff
@@ -0,0 +1,780 @@
+{
```

Comment:
<p>This needs more explanation (what happens in <code>otcp.predict</code>?  ); would also add references to the paper.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/613/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>